denoising_diffusion_pytorch/denoising_diffusion_pytorch.py

```diff
@@ -912,6 +912,7 @@ def __init__(
 
         # FID-score computation
 
+        self.calculate_fid = False
```

Comment:
lol, not a big deal

here you can just set it once like

```python
self.calculate_fid = calculate_fid
if calculate_fid:
```

Reply:
Oh right yeah that went over my head for some reason.

Reply:
Oh right yeah that went over my head for some reason.